src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSliderUI.java

```diff
@@ -150,7 +150,9 @@ public class BasicSliderUI extends SliderUI{
 
     /**
      * Constructs a {@code BasicSliderUI}.
+     * @deprecated This constructor will be removed in future release
```

Comment:
Sounds reasonable enough. It's what happened, isn't it? It's the reason why we're deprecating it and planning to remove it.

```suggestion
     * @deprecated This constructor was added by accident. Do not use it.
     *             This constructor will be removed in a future release.
```

There are comments that say this method shouldn't have been public.

On the other hand, there are quite a few methods and classes in the list of terminally terminated elements which don't say anything at all.

Reply:
Or:

```java
     * @deprecated This constructor will be removed in a future release.
     *             Use {@link #BasicSliderUI(JSlider b)} instead.
```
This is in the gist of the deprecation message for [`SecurityManager`](https://download.java.net/java/early_access/jdk24/docs/api/java.base/java/lang/SecurityManager.html).

Reply:
FWIW, we did something similar to this in JavaFX when we deprecated for removal an accidentally added constructor (an implicit no-arg constructor in our case).

https://github.com/openjdk/jfx/commit/a5ecfb68

```
     * @deprecated This constructor was exposed erroneously and will be removed in the next version.
     * Use {@link #createSelector(String)} instead.
```

If you decide it's worth doing, you might also include a pointer to the other constructor, although since there only is one more, it is pretty obvious for this class.

Reply:
OK...Updated...